<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#color_pushButton {
border-radius: 20px;
color: rgb(255, 102, 178);
font-size: 26px;
border-style: solid;
border-width: 2px;
background-color: rgb(225,224,225);
}


</string>
   </property>
   <widget class="QWidget" name="colorwidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>781</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#colorwidget {
border-radius: 20px;
background-color: rgb(225,224,225);
}

#xyz_groupbox {
color: rgb(255, 102, 178);
font-size: 26px;
border-style: solid;
border-width: 2px;
border-color: rgb(255, 102, 178);
border-radius: 10px;
}

#rgb_groupbox {
color: rgb(255, 102, 178);
font-size: 26px;
border-style: solid;
border-width: 2px;
border-color: rgb(255, 102, 178);
border-radius: 10px;
}

#Lab_groupbox {
color: rgb(255, 102, 178);
font-size: 26px;
border-style: solid;
border-width: 2px;
border-color: rgb(255, 102, 178);
border-radius: 10px;
}

#red_rgb {
color: rgb(255, 102, 178);
}

#hue_hsv {
color: rgb(255, 102, 178);
}

#red_xyz {
color: rgb(255, 102, 178);
}

#green_rgb {
color: rgb(255, 102, 178);
}

#saturation_hsv {
color: rgb(255, 102, 178);
}

#green_xyz {
color: rgb(255, 102, 178);
}

#blue_rgb {
color: rgb(255, 102, 178);
}

#value_hsv {
color: rgb(255, 102, 178);
}

#blue_xyz {
color: rgb(255, 102, 178);
}

</string>
    </property>
    <widget class="QGroupBox" name="rgb_groupbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>RGB</string>
     </property>
     <widget class="QSlider" name="red_rgb_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="blue_rgb_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>124</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="green_rgb_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>87</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="green_rgb">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>89</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="blue_rgb">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>126</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="red_rgb">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>52</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="red_rgb_spinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="green_rgb_spinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>86</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="blue_rgb_spinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>122</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="xyz_groupbox">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>10</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>XYZ</string>
     </property>
     <widget class="QSlider" name="red_xyz_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="blue_xyz_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>124</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="green_xyz_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>87</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="green_xyz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>89</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="blue_xyz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>126</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="red_xyz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>52</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="red_xyz_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="green_xyz_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>86</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="blue_xyz_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>122</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Lab_groupbox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Lab</string>
     </property>
     <widget class="QSlider" name="L_lab_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="b_lab_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>124</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="a_lab_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>87</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="a_lab">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>89</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QLabel" name="b_lab">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>126</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="L_lab">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>52</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>L</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="L_lab_spinBox">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>50</y>
        <width>46</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="b_lab_spinBox">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>124</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="a_lab_spinBox">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>84</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="frame_widget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>270</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_widget {
background-color: rgb(225,224,225);
border-style: solid;
border-width: 3px;
border-color: rgb(210,210,220);
border-radius: 20px;
}

#label {
background-color: white;
border-radius: 14px; 
}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>230</width>
       <height>230</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="color_pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
